self-paced-labs/vertex-ai/vertex-ai-qwikstart/online-retail-clv-3M/trainer/task.py

```diff
@@ -0,0 +1,34 @@
+import os
+import argparse
+
+from trainer import model
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    # Vertex custom container training args. These are set by Vertex AI during training can be overwritten.
```

Comment:
you corrected this sentence in the %%writefile as following, so we can update here too:
    # Vertex custom container training args. These are set by Vertex AI during training but can also be overwritten.

Reply:
Thanks, done.